src/components/tabs/__internal__/tabs-header/tabs-header.style.ts

```diff
@@ -165,27 +99,89 @@ const StyledVerticalTabsWrapper = styled.div<StyledVerticalTabsWrapperProps>`
   flex-direction: column;
 `;
 
-const StyledTabsBottomBorderWrapper = styled.div`
+type StyledWrapperProps = {
+  align: string;
+  position: string;
+  role?: string;
+  extendedLine?: boolean;
+  noRightBorder?: boolean;
+  isInSidebar?: boolean;
+};
+
+type StyledNavigationButtonWrapperProps = {
+  position: string;
+  visible: boolean;
+};
+
+const StyledWrapper = styled.section<StyledWrapperProps>`
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  ${({ extendedLine = true }) =>
+    !extendedLine &&
+    css`
+      width: fit-content;
+    `}
+
+  ${({ position }) => position === "top" && "white-space: nowrap"};
+
+  ${({ align }) =>
+    align === "right" &&
+    css`
+      justify-content: flex-end;
+      text-align: right;
+    `}
+`;
+
+const StyledNavigationButtonWrapper = styled.div<StyledNavigationButtonWrapperProps>`
+  width: 48px;
+  height: 48px;
+
+  ${({ position, visible }) => css`
+    display: ${visible ? "block" : "none"};
+    ${position === "right" && "right: 0; position: absolute;"}
+  `}
+`;
+
+const StyledNavigationButton = styled.button`
   position: absolute;
-  width: 100%;
-  height: auto;
-  bottom: 0;
+  height: 100%;
+  width: 48px;
+  top: 0;
+  align-items: center;
+  border: none;
+  border-radius: 0px;
+  background-color: var(--colorsUtilityMajor050);
+  color: var(--colorsActionMinor500);
+  z-index: 2;
 `;
 
-const StyledTabsBottomBorder = styled.div`
-  position: sticky;
-  bottom: 2px;
-  left: ${outlineWidth};
-  right: ${outlineWidth};
-  height: 2px;
-  background-color: var(--colorsActionMinor100);
+const StyledContainer = styled.div<{ size: string }>`
+  display: flex;
+  padding: 6px 24px 0px;
+  margin: 0;
+  overflow-x: hidden;
+  ${({ size }) => css`
+    height: ${size === "large" ? "54px" : "46px"};
+  `}
+`;
+
+const StyledBottomBorder = styled.div`
```

Comment:
**suggestion**: it looks like all the content in the tab list is currently floating above the grey dividing line, which doesn't match the design specs (see images below - the dividing line has been darkened for clarity):

![indicator-after](https://github.com/user-attachments/assets/313a06c5-ae9b-46e3-8f5a-d8a11b601706)
![validaiton-border-after](https://github.com/user-attachments/assets/f9ebd7b6-92cf-4246-98c1-136a0a65139e)
![navigation-button-above-divider-line](https://github.com/user-attachments/assets/918d8c4d-5b4d-4854-baff-2ccbb66bc739)

I noticed we use relative positioning a fair amount in `Tabs`, particularly to work around the overflow restrictions required to hide overflowing tab titles. Could we adjust the styling to fix this positioning issue?

Reply:
Fixed the floatiness but the DS suggests the focus border [should appear over everything else](https://zeroheight.com/2ccf2b601/p/54468c-tab/t/page-54468c-69590208-28). I'm having issues getting the bottom border to appear given the height of the tab element etc. but I actually think I prefer the new version... 😆 
 
<img width="367" alt="Screenshot 2024-11-14 at 19 18 39" src="https://github.com/user-attachments/assets/b7cd801b-2fc9-46fa-aa16-93890016d010">

<img width="380" alt="Screenshot 2024-11-14 at 19 18 55" src="https://github.com/user-attachments/assets/dca77fd9-4677-431e-ac8e-2437c7f0df5b">